mocks/sync_producer_test.go

```diff
@@ -161,6 +161,12 @@ func TestSyncProducerWithCheckerFunctionForSendMessagesWithoutError(t *testing.T
 		t.Error("No error expected on SendMessages call, found: ", err)
 	}
 
+	for i, msg := range msgs {
+		if int64(i+1) != msg.Offset {
+			t.Errorf("The %dth message should have been assigned offset 1, but got %d", i, msg.Offset)
```

Comment:
is this true `should have been assigned offset 1`?

Reply:
Whoops. It's a typo.